locate-ore.lua

```diff
@@ -132,25 +139,36 @@ local function getOreDescription(ore)
     return str
 end
 
-local args = {...}
-local target = args[1]
+local options, args = {
+    help = false,
+    list = false,
+    show_undiscovered = false
+}, {...}
+
+local positionals = argparse.processArgsGetopt(args, {
+    {'h', 'help', handler=function() options.help = true end},
+    {'l', 'list', handler=function() options.list = true end},
```

Comment:
unless we want `--list` to be an option restricting the output of a command (e.g. `locate-ore hematite --list`), I think we should stick to `list` being a command instead of an option.